bookkeeper-server/src/main/java/org/apache/bookkeeper/tools/cli/commands/client/SimpleTestCommand.java

```diff
@@ -102,10 +101,12 @@ protected void run(BookKeeper bk, Flags flags) throws Exception {
                 }
             }
             LOG.info("{} entries written to ledger {}", flags.numEntries, wh.getId());
-            if (flags.cleanup) {
-                LOG.info("Cleaning up the ledger {}", wh.getId());
-                result(bk.newDeleteLedgerOp().withLedgerId(wh.getId()).execute());
+    
+            try (ReadHandle rh = result(bk.newOpenLedgerOp().withLedgerId(wh.getId()).withDigestType(DigestType.CRC32C)
+                    .withPassword(new byte[0]).execute())) {
+                rh.read(0, flags.numEntries);
```

Comment:
Would you please assert the result of rh.read()?

Reply:
Fine, the write data is an empty array. I will change the written data and verify it.